distribution/docs/src/main/resources/content/_coreConcepts/high-availability-intro.adoc

```diff
@@ -0,0 +1,56 @@
+:type: coreConcept
+:status: published
+:title: Introduction to High Availability
+:order: 10
+
+${branding} can be made highly available. In this context, High Availability is defined as the ability for ${branding} to be continuously operational with very little down time.
+
+In a Highly Available Cluster, ${branding} has failover capabilities when a ${branding} node fails.
+
+[NOTE]
+====
+The word "node", from a High Availability perspective, is one of the two ${branding} systems running within the Highly Available Cluster.
+Though there are multiple systems running with the Highly Available Cluster, it is still considered a single ${branding} from a user's
+perspective or from other ${branding}s' perspectives.
+====
+
+This setup consists of a Solr Cloud instance, 2 ${branding} nodes connected to that Solr Cloud, and a failover proxy that sits in front of those 2 nodes.
```

Comment:
❓ `2 (or more)`?

Reply:
No, right now HA only means 2 nodes.

Reply:
❓ I thought it was possible to scale to more nodes if you configured it that way. Is that possible?

Reply:
It's really dependent on the failover proxy used and how it's configured. In theory, it's possible but it's not when we tested so we won't be documenting that for now.